src/corehost/cli/setup.cmake

```diff
@@ -71,5 +71,5 @@ if(${CLI_CMAKE_PKG_RID} STREQUAL "")
     message(FATAL_ERROR "A minimum supported package rid is not specified (ex: win7-x86 or ubuntu.14.04-x64, osx.10.10-x64, rhel.7-x64)")
 endif()
 
-add_definitions(-DHOST_POLICY_PKG_NAME="Microsoft.NETCore.DotNetHostPolicy")
+add_definitions(-DHOST_POLICY_PKG_NAME="runtime.${CLI_CMAKE_PKG_RID}.Microsoft.NETCore.DotNetHostPolicy")
```

Comment:
Sorry, this is a bug fix. The architecture discussion prompted me to think about this.